src/WhatToCook.Application/Services/RecipeService.cs

```diff
@@ -19,24 +19,38 @@ public RecipeService(IRecipesRepository recipesRepository, ILogger<RecipeService
 
     public async Task<Recipe> Create(RecipeRequest request, string imagesDirectory)
     {
-        var imageInfo = new ImageInfo(request.Image, Guid.NewGuid().ToString(), imagesDirectory);
-        var imagePath = await _recipesRepository.SaveImage(imageInfo);
+        string imagePath = "";
+        if (request.Image.Length > 0)
+        {
+            var imageInfo = new ImageInfo(request.Image, Guid.NewGuid().ToString(), imagesDirectory);
+            imagePath = await _recipesRepository.SaveImage(imageInfo);
+        }
+        else
+        {
+            imagePath = $"Images/default_image.png";
+        }
+
         var ingredients = request.Ingredients.Select(ingredient => new Ingredient(ingredient)).ToList();
         var recipe = new Recipe
         (
             name: request.Name,
             description: request.PreparationDescription,
             timeToPrepare: request.TimeToPrepare,
             ingredients: ingredients,
-            statistics : new Statistics(),
+            statistics: new Statistics(),
             image: imagePath,
-            plansOfMeals: new List<PlanOfMeals>()
+            plansOfMeals: new List<RecipePlanOfMeals>()
```

Comment:
This one should be skipped here. So the constructor should not require this at all I think. Just initialize it inside of the Recipe with empty